labs/hyperledger-signare.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: Signare
+parent: Labs
+---
+# Lab Name
+
+Hyperledger Signare
+
+# Short Description
+
+Signare, a Hyperledger Lab, is an enterprise grade digital signing solution for DLT related applications and Ethereum clients.
+
+# Scope of Lab
+
+A security concern shared by most users of DLT applications is "keeping their private key private". In the enterprise space FIPS 140 is often used to inform institutions of how they must manage their private keys. Specifically, FIPS 140-2 Level 2 adds requirements for physical tamper-evidence (and/or tamper-resistance) and role-based authentication, which necessitates the use of an HSM or Cloud HSM. 
+
+The purpose of Hyperledger Signare is to provide a FIPS 140-2 Level 2 compliant signing solution for enterprise applications where various HSM and Cloud HSM vendors will be supported via plugins. Hyperledger Signare also provides role-based access controlled interfaces to solve multiple usecases, such as signing Ethereum transactions and for blockchain clients such as Hyperledger Besu to store keys in an HSM or Cloud HSM.
+
+
+# Initial Committers
+
+https://github.com/nano-adhara
+https://github.com/chookly314
+https://github.com/Jserrano27
+https://github.com/gynura
+https://github.com/ArturoGarciaRegueiro
+https://github.com/mkrielza
+https://github.com/coeniebeyers
+
+
+# Sponsor
+ 
+- Susumu Toriumi (susumu.toriumi@datachain.jp) - Maintainer, YUI
```

Comment:
Can you please add Github of the sponsor.

Reply:
@storiumi : Can you confirm sponsorship?

Reply:
@tkuhrt (cc. @coeniebeyers ) Confirmed. Happy to support the lab.